saltstack/base/salt/prerequisites/usr/bin/ccmv2-helper.sh

```diff
@@ -4,6 +4,13 @@ setup_ccmv2() {
   : ${CCM_V2_AGENT_KEY_ID:? required}
   : ${CCM_V2_AGENT_BACKEND_ID_PREFIX:? required}
 
+ACCESS_KEY_PATH=$(touch /etc/ccmv2/access_key)
```

Comment:
This results in an empty string because `touch` does not print anything to stdout.